card_live_dashboard/layouts/__init__.py

```diff
@@ -283,6 +283,21 @@ def figures_layout(figures_dict: Dict[str, go.Figure]):
                                      value_color='default'
                                  ),
                                  ),
+            single_figure_layout(title='RGI intersections',
+                                 description=['Patterns of co-occurrence of the above RGI result type across selected genomes'],
```

Comment:
Maybe change **above RGI result type** to **selected RGI result type** since the selection is no-longer done above this text.

Reply:
Fixed!